#!/usr/bin/env python3
PACKAGE = "two_mats_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("thresh",                  int_t,    0, "Thresh Value", 127, 0, 255)
gen.add("enable_drive",            bool_t,   0, "Enable",       False)

exit(gen.generate(PACKAGE, "two_mats", "TwoMats"))
# The 3rd param "StopAtEdge" must be this filename
